towhee/functional/data_collection.py

```diff
@@ -459,40 +460,66 @@ def range(*arg, **kws):
         """
         return DataCollection(range(*arg, **kws))
 
-    def batch(self, size, drop_tail=False) -> 'DataCollection':
+    def batch(self, size, drop_tail=False, raw=True):
```

Comment:
can we just put the entities into a list here, and don't deal with the fields.

Reply:
I have tried it but not work. If we yield a list of entities will got:
```bash
~/workspace/git/towhee/towhee/engine/factory.py in <listcomp>(.0)
     73     def __apply__(self, *arg, **kws):
     74         if isinstance(self._index[0], tuple): # Multi inputs.
---> 75             args = [getattr(arg[0], x) for x in self._index[0]]
     76         else: # Single input.
     77             args = [getattr(arg[0], self._index[0])]

AttributeError: 'list' object has no attribute 'path'
```
and if yield DataCollection directly, the args in op is like `(<towhee.hparam.hyperparameter.wrapper object at 0x7f95031df460>, <towhee.hparam.hyperparameter.wrapper object at 0x7f95031df520>)`.

And the example code is follows:
```python
import towhee
from towhee import connector

milvus_collection = connector.milvus(uri='tcp://localhost:19530/reverse_image_search')            
                    

with milvus_collection:
    (
        towhee.glob['path']('/Users/chenshiyu/workspace/data/pic/test/*.jpg')
              .image_decode['path', 'img']()
              .image_embedding.timm['img', 'vec'](model_name='resnet50')
              .runas_op['path', 'path'](func=lambda path: abs(hash(path)) % (10 ** 8))
              .batch(3)
              .ann_insert[('path', 'vec'), 'res'](ann_index=milvus_collection)
              .run()
    )
```

Reply:
@GuoRentong We run into this problem angin, the behavior of batch  has not been defined for entities. I'll open another issue for this problem.